### PR TITLE
Add `prevent_alert_auto_close` option for check plugins

### PR DESCRIPTION
**Pull request by mechairoi**
If `disable_auto_close` option is true, alerts of the check plugin are no longer closed automatically.
This option make mackerel-agent not report `OK` status of the plugin to the server.

`disable_auto_close = true` option is not available in combination with `max_check_attempts > 1` option.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -136,21 +137,28 @@ type CheckPlugin struct {
...
 		Command:              pconf.Command,
 		CommandArgs:          pconf.CommandArgs,
 		User:                 pconf.User,
 		NotificationInterval: pconf.NotificationInterval,
 		CheckInterval:        pconf.CheckInterval,
 		MaxCheckAttempts:     pconf.MaxCheckAttempts,
-	}, nil
+	}
+	if pconf.DisableAutoClose == nil {
+		plugin.DisableAutoClose = false
```
Comment (astj) [inline config/config.go]: [nit]
You can omit explicitly `false` assignment, since it's treated as `false` when it's not assigned.

Comment (astj) [inline config/config.go]: Using `bool` (not `*bool`) in `pconf.DisableAutoClose` can simplify this assignment?

Comment (mattn) [inline config/config.go]: If you expect false in default, EnableAutoClose is better. (This is not strong opinion)

Comment (mechairoi) [inline config/config.go]: I expect `EnableAutoClose = true`(`DisableAutoClose = false`) in default for backward compatibility.

Comment (itchyny) [inline config/config.go]: BTW user cannot tell _close what?_. Some suggestions: `keep_alert_open`, `prevent_alert_auto_close`.

Comment (stefafafan) [inline config/config.go]: I personally like `prevent_alert_auto_close` , it seems simple and clear.

**Comment by astj** [comment]
LGTM!

**Pull request merged**